seed/lib/superperms/orgs/permissions.py

```diff
@@ -55,21 +55,43 @@ def get_org_or_id(dictlike):
 
 
 def get_org_id(request):
-    """Get org id from request"""
-    org_id = get_org_or_id(request.query_params)
-    if not org_id:
+    """Extract the ``organization_id`` regardless of HTTP method type."""
```

Comment:
updated method by merging with the _get_org_id func from decorators.py. Also tweaked the sequence of where we look for IDs.